contracts/GateSeal.vy

```diff
@@ -139,12 +157,45 @@ def get_expiry_timestamp() -> uint256:
     return self.expiry_timestamp
 
 
+@external
+@view
+def get_dao() -> address:
+    return DAO
+
+
+@external
+@view
+def get_extension_duration_seconds() -> uint256:
+    return EXTENSION_DURATION_SECONDS
+
+
+@external
+@view
+def get_extensions_remaining() -> uint256:
+    return self.extensions_remaining
+
+
 @external
 @view
 def is_expired() -> bool:
     return self._is_expired()
 
 
+@external
+def extend():
+    """
+    @notice Extend the GateSeal expiry.
+    @dev    Can be called only by DAO while the seal hasn't been used and not expired.
+    """
+    assert msg.sender == DAO, SENDER_NOT_DAO
```

Comment:
Wouldn't it make more sense for the committee to do it? By shifting this responsibility to the committee, we can ensure that the committee is still in tact (e.g. no member lost their keys)